doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -135,6 +135,8 @@ The well known service meshes are "application service meshes" that address and
 <a name="3.2.3"></a>
 ### 3.2.3 Container Storage Services
 
+Since its 1.13 version Kubernetes supports Container Storage Interface (CSI) in production and in-tree volume plugins are moved out from the Kubernetes repository. It is advised that in an RA2 compliant cluster the storage is managed with a [CSI driver](https://kubernetes-csi.github.io/docs/drivers.html) instead of an in-tree volume plugin.
```

Comment:
Suggest removing the second sentence "It is advised that in an RA2 compliant cluster the storage is managed with a [CSI driver](https://kubernetes-csi.github.io/docs/drivers.html) instead of an in-tree volume plugin." - we can add in any specification into chapter04.md

Reply:
agree with Tom

Reply:
Fixed now.